src/components/layouts/base/menues/connectedDropdown.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-list-item-group :style="{ backgroundColor: 'white' }" active-class="text--accent-4">
+    <v-list-item to="/">
+      <v-list-item-icon>
+        <v-icon>mdi-home</v-icon>
+      </v-list-item-icon>
+      <v-list-item-title>Home</v-list-item-title>
+    </v-list-item>
+
+    <v-list-item to="/myActivity">
+      <v-list-item-icon>
+        <v-icon>mdi-chart-bar</v-icon>
+      </v-list-item-icon>
+      <v-list-item-title>Dashboard</v-list-item-title>
+    </v-list-item>
+
+    <v-list-item to="/supplyBorrow">
+      <v-list-item-icon>
+        <v-icon>mdi-bank</v-icon>
+      </v-list-item-icon>
+      <v-list-item-title>Supply/Borrow</v-list-item-title>
+    </v-list-item>
+
+    <v-list-item to="/status">
+      <v-list-item-icon>
+        <v-icon>mdi-frequently-asked-questions</v-icon>
```

Comment:
```suggestion
        <v-icon>mdi-view-list-outline</v-icon>
```
faq icon already used that

Reply:
I made an update with the store icon, if you don't agree you can suggest one from the following list https://material.io/resources/icons/?icon=store&style=baseline

Reply:
sounds good